app/Http/Controllers/Login.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Helpers\Cookie;
+use App\Helpers\Redirect;
+use App\Libraries\Controller;
+use App\Libraries\Database;
+use App\Models\User;
+use App\Http\Middleware\IsAuthenticatedMiddleware;
+
+class Login extends Controller
+{
+    public function index()
+    {
+        if (IsAuthenticatedMiddleware::handle()) Redirect::To('/dashboard');
+
+        if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
You can create a class Server and have a method that will check for the different method types to simplify this logic.